### PR TITLE
docs: Name Change

### PR DESCRIPTION
**Pull request by connerleverett**
Change Geocortex -> VertiGIS Studio and geocortex.com -> vertigisstudio.com

### CODE REVIEW

**Comment by connerleverett** [comment]
To be merged after Feb 14th 2022

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 [![CI/CD](https://github.com/geocortex/workflow-activities-elementsxs/workflows/CI/CD/badge.svg)](https://github.com/geocortex/workflow-activities-elementsxs/actions) 
 [![npm](https://img.shields.io/npm/v/@geocortex/workflow-activities-elementsxs)](https://www.npmjs.com/package/@geocortex/workflow-activities-elementsxs)
 
-This project contains activities for interacting with the [Elements XS REST API](https://salesapi.novotx.com/) in a [Geocortex Workflow](https://www.geocortex.com/products/geocortex-workflow/).
+This project contains activities for interacting with the [Elements XS REST API](https://salesapi.novotx.com/) in a [VertiGIS Studio Workflow](https://www.vertigisstudio.com/products/geocortex-workflow/).
```
Comment (kenlyon) [inline README.md]: Should the product name change here?

**Review by kenlyon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (connerleverett) [reply README.md]: Updated.

**Review by rcooney — COMMENTED** [review]
The changes all look good. We just need to wait until https://apps.vertigisstudio.com/workflow/designer/ is live.

**Review by kenlyon — APPROVED** [review]

**Pull request merged**